### DISCUSSION TITLE
Setup appium

### DISCUSSION THREAD

**Discussion by oanhinfo** [webdriverio/webdriverio#13049] [Community]
Is there any documents or guideline to walk me through the set up of using appium for native mobile app? I find a video on youtube but i wonder why this guy know how to set thing up.

**Comment by christian-bromann** [comment]
I would recommend to get familiar with Appium [docs](https://appium.io/docs/en/latest/intro/) use the [Appium service](https://webdriver.io/docs/appium-service) and use Appium Doctor to check if everything is set up right.
